### PR TITLE
Fix percentile computation when numpy isn't installed

### PR DESCRIPTION
**Pull request by cpennington**
@jimabramson this should fix the errors you were having when you didn't have numpy installed.

### CODE REVIEW

File: locust/stats.py [hunk 1]
```diff
@@ -483,7 +483,7 @@ def get_response_time_percentiles(self, percentiles=PERCENTILES):
             target_percentiles = sorted(percentiles, reverse=True)
             for response_time, count in sorted(self.response_times.iteritems(), reverse=True):
                 processed_count += count
-                while (self.num_requests - processed_count) <= target_percentiles[0]:
+                while target_percentiles and float((self.num_requests - processed_count)) / self.num_requests <= target_percentiles[0]:
```
Comment (jimabramson) [inline locust/stats.py]: why the double parens?

**Comment by cpennington** [comment]
@jimabramson @cgoldberg @doctoryes: Review?

**Comment by jimabramson** [comment]
:+1:  manually verified that this fixes the error.

File: locust/stats.py [hunk 2]
```diff
@@ -483,7 +483,7 @@ def get_response_time_percentiles(self, percentiles=PERCENTILES):
             target_percentiles = sorted(percentiles, reverse=True)
             for response_time, count in sorted(self.response_times.iteritems(), reverse=True):
                 processed_count += count
-                while (self.num_requests - processed_count) <= target_percentiles[0]:
+                while target_percentiles and float(self.num_requests - processed_count) / self.num_requests <= target_percentiles[0]:
```
Comment (doctoryes) [inline locust/stats.py]: Nit: Confusing - and no comments.

**Comment by doctoryes** [comment]
:+1:

**Pull request merged**